spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/catalog/GravitinoCatalog.java

```diff
@@ -147,24 +148,30 @@ public Table createTable(
                   partitionings,
                   distributionAndSortOrdersInfo.getDistribution(),
                   distributionAndSortOrdersInfo.getSortOrders());
-      return gravitinoAdaptor.createSparkTable(ident, table, sparkCatalog, propertiesConverter);
+      Table sparkTable = sparkCatalog.loadTable(ident);
```

Comment:
What is the purpose of this step?

Reply:
![image](https://github.com/datastrato/gravitino/assets/94670132/c9571d6f-afcc-409c-bef1-8b4ddc805227)

To get the parameters of the super method.